public/pages/CreateTrigger/utils/constants.js

```diff
@@ -32,6 +32,11 @@ export const FORMIK_INITIAL_ACTION_VALUES = {
     lang: 'mustache',
     source: DEFAULT_MESSAGE_SOURCE,
   },
+  throttle_enabled: true,
+  throttle: {
+    value: 10,
+    unit: 'MINUTES'
```

Comment:
np. Could you add comment mentioning we don't have UI input for this field yet so this value will be default for all the inputs.

Reply:
Sure, will add comment